src/components/ProfileHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import { TiArrowBack } from "react-icons/ti"
```

Comment:
Try to make use of material-ui icons while we are still using their tools so that icons are the same across all platforms.

Reply:
Will be done in the redesign